tests/custom_networks.py

```diff
@@ -0,0 +1,36 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from hivemind.server.layers.custom_experts import register_expert_class
+
+sample_input = lambda batch_size, hidden_dim : torch.empty((batch_size, hidden_dim))
+@register_expert_class('perceptron', sample_input)
+class MultilayerPerceptron(nn.Module):
+    def __init__(self, hidden_dim, num_classes=10):
+        super(MultilayerPerceptron, self).__init__()
+        self.layer1 = nn.Linear(hidden_dim, 2 * hidden_dim)
+        self.layer2 = nn.Linear(2 * hidden_dim, 2 * hidden_dim)
+        self.layer3 = nn.Linear(2 * hidden_dim, num_classes)
+
+    def forward(self, x):
+        x = F.relu(self.layer1(x))
+        x = F.relu(self.layer2(x))
+        x = self.layer3(x)
+        return x
```

Comment:
I agree about multi-input and loading from separate files, but then the perceptron test isn't necessary, is it?

Reply:
Nevermind, I misunderstood the last part about several custom experts. Thank you!